src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
+    }
+
+    @Override
+    public Boolean delete(Long id) {
```

Comment:
Consider changing the return type of the `delete` method to `void`. Returning a boolean might not be necessary if the method is expected to always succeed or throw an exception otherwise.